highfidelity/hifi
# Entity properties - new draggable numbers

Replace number and slider inputs in the entity properties window with a draggable number field per Alexia's design.

Addresses:
https://highfidelity.manuscript.com/f/cases/19062

Test Plan:
- Enter Interface in desktop mode into any domain
- Open Create app and select or create any entity and view the Properties tab
- Verify the step up/down arrow buttons on the right side of any property number field are now replaced with left and right arrows on each side
- Verify when hovering over the number text (but not over the arrow buttons) that a left/right resize cursor appears
- Verify if holding mouse down on the number text itself (but not over the arrow buttons) and dragging left before releasing mouse that it decreases the number and if dragging right that it increases the number
- Verify if clicking on the number text itself (but not over the arrow buttons) and letting go without moving the mouse that the number field maintains the same value and switches to a state with no step arrows, the text is highlighted, and the text can be edited via keyboard. Verify once clicking away from the number field that it returns to the previous state with arrows and that the number is still exactly the same value.
- Verify all forms of number fields are replaced with this new draggable number and perform as expected which includes regular numbers such as Parent Joint Index and Lifespan, sliders such as particle Size/Alpha/Spin properties, vectors such as Position, Rotation, and Dimension
